core/build/src/lib.rs

```diff
@@ -43,6 +43,10 @@ pub fn normalize_item_name(name: &str) -> Option<String> {
         name.push_str("_t");
     }
 
+    // Some types come through as `_sgx_thread_mutex_attr_t` which map to a real
+    // type of `sgx_thread_mutexattr_t`
+    let mut name = name.replace("_attr_", "attr_");
```

Comment:
I was a bit nervous about false-positives here but to be fair if there was any sort of match to this pattern which shouldn't get matched then the code would just fail to compile rather than blowing up at runtime so it's probably fine.

Reply:
yah, it's an interesting balance of trying to be programatic and concise vs calling out every type name.